k8s/openebs-monitoring/README.md

```diff
@@ -1,39 +1,82 @@
 ## Overview
-* [Prometheus](https://github.com/prometheus/prometheus) is a monitoring tool  which is used to monitor OpenEBS volumes. There is already a lot of exporters like [node-exporter](https://github.com/prometheus/node_exporter) are available which can be used to gather the various metrics. 
-* This subdir has two files
-    - prometheus.yaml
-    - prometheus-operator.yaml
-* prometheus.yaml :  This is the configuration file for prometheus, which can be used by prometheus to get the targets (Where to collect metrics?) and it displays the collected metrics in Prometheus expression browser.
-* prometheus-operator.yaml : This is the operator file for prometheus which is used to launch prometheus and it's components on [Kubernetes](https://github.com/kubernetes/kubernetes) cluster.
+* OpenEBS use [Prometheus](https://github.com/prometheus/prometheus) to  monitor Volumes . We use [node-exporter](https://github.com/prometheus/node_exporter) to gather nodes metrics, [cadviser](https://github.com/google/cadvisor) for containers metrics and maya-agent to gather OpenEBS volume's metrics. 
+* This sub directory has following files:
+├── openebs-monitoring
+│   ├── alertmanager.yaml
+│   ├── configs
+│   │   ├── alertmanager-config.yaml
+│   │   ├── alertmanager-templates.yaml
+│   │   ├── prometheus-alert-rules.yaml
+│   │   ├── prometheus-config.yaml
+│   │   └── prometheus-env.yaml
+│   ├── grafana-operator.yaml
+│   ├── prometheus-operator.yaml
+│   └── README.md
+
+    * alertmanager.yaml : This is for creating deployment and service of alertmanager.
+    * alertmanager-config : This is configuration file for alertmanager, used to load templates and to set alerts in slack. This can be configured to to set alerts at various platforms like e-mail, slack etc.
+    * alertmanager-templates.yaml : This is used to customize notifications send to slack.
+    * prometheus-alert-rules.yaml: Rules defined for alerting is passed  in prometheus-deployment.
+    * prometheus-config.yaml :  This is  configuration file for prometheus, which can be used by prometheus to receive the targets and it displays the collected metrics in the Prometheus expression browser.
+    * prometheus-env: This is to store all the necessary runtime arguments to Prometheus.
+    * grafana-operator.yaml : This file is to create deployment and start service of grafana for various monitoring and analytics.
```

Comment:
grafana-operator.yaml : = grafana-operator.yaml:
This file is to create = This file creates deployment and starts grafana service for various monitoring and analytics processes?